config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml

```diff
@@ -89,6 +89,48 @@ periodics:
           memory: "9000Mi"
           cpu: 2000m
 
+- name: ci-kubernetes-e2e-kubeadm-kinder-eks-canary-1-27
+  cluster: eks-prow-build-cluster
+  interval: 1h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kubeadm
+    testgrid-tab-name: kubeadm-kinder-eks-canary-1-27
```

Comment:
@ameukam @dims 

i just saw this job in the kubeadm dashboard.
this kubeadm-kinder.yaml file is actually generated and this will be overwritten once we push new changes.

it has to be included in the config here:
https://github.com/kubernetes/kubeadm/tree/main/kinder/ci/tools/update-workflows

or perhaps just copy-pasted into a different folder / owning sig / dashboard.

cc @SataQiu @pacoxu